server/handlers/getAdmins.js

```diff
@@ -0,0 +1,23 @@
+const passport = require('passport');
+
+const getAdmins = (req, res, next) => {
+  passport.authenticate('jwt', { session: false }, (err, user, info) => {
+    if (err) {
+      throw new Error(err);
+    }
+    if (info !== undefined) {
```

Comment:
if (info) would work?

Reply:
Agreed! Fixed in `jwt-authentication-login-page` branch